frontend/app/helpers/path-helper.js

```diff
@@ -217,26 +217,6 @@ module.exports = function() {
       return PathHelper.apiWorkPackagesPath() + '/column_sums';
     },
 
```

Comment:
@NobodysNightmare it is getting worse when you look at the way the information is used:
https://github.com/NobodysNightmare/openproject/blob/feature/remove_apiv2_access/frontend/app/timelines/services/timeline-loader-service.js#L665
The sad truth is: "We have a PathHelper and use it in timelines to get the prefix which we use to construct paths"
You feel like touching timelines?